.github/workflows/patch-images.yaml

```diff
@@ -0,0 +1,69 @@
+on:
+  schedule:
+    - cron: "0 0 * * *" # nightly
+  pull_request:
+    branches:
+      - main
```

Comment:
why do we need this to be run as part of every PR?

Reply:
we don't, removed!